sktime/utils/validation/forecasting.py

```diff
@@ -328,13 +329,22 @@ def check_scoring(scoring):
     ----------
     TypeError
         if object is not callable from current scope.
+    NotImplementedError
+        if metric requires y_pred_benchmark to be passed
     """
     # Note symmetric=True is default arg for MeanAbsolutePercentageError
     from sktime.performance_metrics.forecasting import MeanAbsolutePercentageError
 
     if scoring is None:
         return MeanAbsolutePercentageError()
 
+    if _has_tag(scoring, "requires-y-pred-benchmark") and allow_y_pred_benchmark:
```

Comment:
Just a comment: Python uses some shortcuts to evaluate if statements, so if the the first one evaluate to False and there is an AND operator it stops and returns False, so it may be faster to put cheaper evaluations first, but it doesn't really matter here since both conditions are cheap (https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not).

Reply:
Good point. I'll keep that in mind.